Description: Use Unix style line endings in dkms.conf
 Use Unix style line endings in dkms.conf instead of windows styled ones. This
 is required for DKMS to correctly parse it.
 .
 tuxedo-yt6801 (1.0.28-1) jammy; urgency=medium
 .
   * Initial release.
Author: Werner Sembach <tux@tuxedocomputers.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2024-04-09

--- tuxedo-yt6801-1.0.28.orig/dkms.conf
+++ tuxedo-yt6801-1.0.28/dkms.conf
@@ -1,8 +1,8 @@
-PACKAGE_NAME="yt6801"
-PACKAGE_VERSION="1.0.28"
-CLEAN="make clean"
-MAKE[0]="make"
-BUILT_MODULE_NAME[0]="yt6801"
-DEST_MODULE_LOCATION[0]="/kernel/drivers/net/ethernet/motorcomm"
-AUTOINSTALL="yes"
-REMAKE_INITRD="yes"
+PACKAGE_NAME="yt6801"
+PACKAGE_VERSION="1.0.28"
+CLEAN="make clean"
+MAKE[0]="make"
+BUILT_MODULE_NAME[0]="yt6801"
+DEST_MODULE_LOCATION[0]="/kernel/drivers/net/ethernet/motorcomm"
+AUTOINSTALL="yes"
+REMAKE_INITRD="yes"
